src/actions/websocket.js

```diff
@@ -602,7 +613,7 @@ function handleHelloEvent(msg) {
     const serverVersion = msg.data.server_version;
     if (serverVersion && Client4.serverVersion !== serverVersion) {
         Client4.serverVersion = serverVersion;
-        EventEmitter.emit(General.CONFIG_CHANGED, serverVersion);
+        EventEmitter.emit(General.SERVER_VERSION_CHANGED, serverVersion);
```

Comment:
if you do this change here, then you'll need to make some changes in [mobile](https://github.com/mattermost/mattermost-mobile/blob/master/app/mattermost.js#L82) as well as this was used to request the new config from the server, you have to assume that the websocket is not connected and you already have a config loaded into the offline store.

when the config or the license changed this version used to change as well so with any new request this would trigger that event thus we are requesting the new config from the server.

makes sense?

Reply:
I had already made some changes to mobile, but was waiting for this PR to update the `yarn.lock`. Of course, reviewing this doesn't make sense without the context of that PR, so I've pushed it up here: https://github.com/mattermost/mattermost-mobile/pull/1469.

I'd love your feedback on the comments in that PR. Effectively, this means /no/ changes in the short term, since we can't assume we're connected to a server which will emit these websockets, at least not without server version parsing or some other signal. It didn't seem like a bad thing to keep the existing refresh alongside the websocket in all cases, at least until we move past supporting these servers.